reference/shipping.v3.yml

```
@@ -185,16 +185,161 @@ paths:
                   meta:
                     type: object
                     properties: {}
+        '400':
+          description: Bad Request. Input is invalid.
+          content:
+            application/json:
+              schema:
+                description: ''
+                type: object
+                properties:
+                  status:
+                    type: number
+                  title:
+                    type: string
+                    minLength: 1
+                  type:
+                    type: string
+                    minLength: 1
+                  detail:
+                    type: string
+                    minLength: 1
+              example:
+                status: 400
+                title: Input is invalid
+                type: 'https://developer.bigcommerce.com/api-docs/getting-started/api-status-codes'
+                detail: Syntax error
+        '422':
+          description: The request body does not meet the specification.
+          content:
+            application/json:
+              schema:
+                description: ''
+                type: object
+                properties:
+                  status:
+                    type: number
+                  title:
+                    type: string
+                    minLength: 1
+                  type:
+                    type: string
+                    minLength: 1
+              example:
+                status: 422
+                title: Input is invalid
+                type: 'https://developer.bigcommerce.com/api-docs/getting-started/api-status-codes'
       summary: Update Shipping Settings
       description: |-
-        Updates shipping settings.
+        Updates the global-level shipping settings.
       tags:
         - Shipping Settings
       requestBody:
         content:
           application/json:
             schema:
               $ref: '#/components/schemas/shippingSettings'
+  '/shipping/settings/channels/{channel_id}':
+    parameters:
+      - $ref: '#/components/parameters/Accept'
+      - name: channel_id
+        in: path
+        schema:
+          type: string
+        description: Channel ID
+        required: true
+
+    get:
+      operationId: getChannelShippingSettings
+      responses:
+        '200':
+          description: ''
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    $ref: '#/components/schemas/channelShippingSettings'
+                  meta:
+                    type: object
+                    properties: {}
+      description: |-
+        Returns shipping settings for a specific channel.
+      tags:
+        - Shipping Settings
+      summary: Get Shipping Settings per channel
```

```suggestion
      summary: Get Shipping Settings per Channel
```

Looks like we have a capital for `Channel` in the customers documentation. Seems legit.

---

updated 👍